### ISSUE TITLE
I would like to ask how can I see the locked time after voting by edg?

### ISSUE THREAD

**Issue by Bestzeno** [polkadot-js/apps#3441]
![1111](https://user-images.githubusercontent.com/36215225/90584170-92de2500-e204-11ea-9099-704701e1398e.png)
My edg has been locked in the account and cannot be transferred out since voting

**Comment by jacogr** [comment]
If you are willing to share this specific address can take a look at the chain state. I have not seen an “everything” lock with democracy before.

**Comment by Bestzeno** [comment]
No problem, I just saw your reply today.

**Comment by Bestzeno** [comment]
n6auQdGvhYY5ggn2buBJn9sH6bdFJJk9KoVKjBombEnRGD7

**Comment by Bestzeno** [comment]
Hearing from others, it seems that after edg participated in the voting, there is a rule that it must be frozen for 28 days, but I don't see how many days can be unlocked.

**Comment by jacogr** [comment]
So there is good news and bad news. the good news is that these locks will be auto-cleared. The bad news is that until the next Edgeware runtime upgrade, there is not anything that can be done UI-wise. Unlike the current version of Substrate the older version in-use (at this moment) by Edgeware does auto-lock clearing and tracks it internally. So ou can see -

- locks being available
- when the locks started

But not -

- when locks will be cleared

![image](https://user-images.githubusercontent.com/1424473/90868155-b76a0680-e396-11ea-9039-0118f0d076eb.png)

So at this point, this is a "can't fix, the info is not there" and actually a "won't fix since the underlying issue disappears after Edgeware upgrades". (It is really tricky to generally support old and new consistently, at this point the older version just works very differently with less actionable information)

TL;DR You will have more info going forward, but as of now it is a bit of a black box.
